github-build.mjs

```diff
@@ -0,0 +1,58 @@
+import pkg from './package.json'
+import Builder from 'alpheios-node-build'
+import generateBuildInfo from './node_modules/alpheios-node-build/dist/support/build-info.mjs'
+import { execFileSync, execSync } from 'child_process'
+import * as core from '@actions/core'
+
+(async function() {
+  const buildDT = Date.now()
+  const buildInfo = generateBuildInfo(buildDT)
+  console.log(`Starting build ${buildInfo.name}`)
+
+  const baseVersion = pkg.version.split('-')[0]
+  const version = `${baseVersion}-${buildInfo.name}`
+  console.log(`Setting a package version to ${version}`)
+  let output
+  try {
+    if (buildInfo.branch === 'qa' || buildInfo.branch === 'production') {
+      console.info(`Installing alpheios-core`)
```

Comment:
Can we see console messages in GitHub Actions? Are those messages ever needed for GitHub?

Reply:
We can see them. Although these console messages aren't needed for GitHub, I think they are helpful to keep for debugging.